Update give-mailbox-permissions-to-another-user.md (#3762)
MicrosoftDocs/microsoft-365-docs: microsoft-365/admin/add-users/give-mailbox-permissions-to-another-user.md

6. Select **Save**.

> [!NOTE]

> > Read and Modify access is also called as Fullaccess permissions when granting access through Exchange admin center. Full access does not grant **Send as** or **Send on Behalf**  permission.

Reviewer: ```suggestion
> Read and Manage permissions are called Full Access permission when granting access through the Exchange admin center. Full Access permission does not grant **Send as** or **Send on Behalf**  permissions.
```
